content/docs/context.md

```diff
@@ -4,39 +4,83 @@ title: Context
 permalink: docs/context.html
 ---
 
-> Note:
->
-> `React.PropTypes` has moved into a different package since React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types) to define `contextTypes`.
->
->We provide [a codemod script](/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes) to automate the conversion.
+Context provides a way to pass data through the component tree without having to pass props down manually at every level.
+
+- [API](#api)
+  - [React.createContext](#reactcreatecontext)
+  - [Provider](#provider)
+  - [Consumer](#consumer)
+- [Examples](#examples)
+  - [Static Context](#static-context)
+  - [Dynamic Context](#dynamic-context)
+- [Motivation](#motivation)
+- [Legacy API](#legacy-api)
+
+## API
+
+### `React.createContext`
+
+```js
+const {Provider, Consumer} = React.createContext([default]);
```

Comment:
Why `[default]` ? (Why the array syntax?) Suggestion:

```js
const {Provider, Consumer} = React.createContext(defaultValue);
```

Reply:
I was thinking [] indicated optional arguments. Replacing it with a text note is fine; it's not a real type signature or anything.

Reply:
Gotcha. I don't think thats' what "[]" implies to me. 👍